CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.29.0
```

Comment:
The challenge is that we include the timestamps as well, which means we can't really know what the time will be until it's published - this is probably something we should figure out how to automate.

But, I can definitely add all the previously released dates :)